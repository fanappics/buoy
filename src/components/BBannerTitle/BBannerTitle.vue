<template>
  <span class="b-banner-title">
    <slot></slot>
  </span>
</template>

<script>
export default {
  name: 'b-banner-title',
  props: {
    primary: Boolean,
    secondary: Boolean,
    tirtiary: Boolean
  },
  computed: {
    classes () {
      return {
        'b-banner-title': true,
        'primary': this.primary,
        'secondary': this.secondary,
        'tirtiary': this.tirtiary
      }
    }
  }
}
</script>

<style lang="stylus" scoped>
  span
    &.b-banner-title
      a
        text-decoration none
        font-size 36px
        font-weight 600
        color #fff

  span
    &.b-banner-title
      &.secondary
        a
          color #000
</style>
