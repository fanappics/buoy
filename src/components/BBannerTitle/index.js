import BBannerTitle from './BBannerTitle.vue'

/* istanbul ignore next */
BBannerTitle.install = function install (Vue) {
  Vue.component(BBannerTitle.name, BBannerTitle)
}

export default BBannerTitle
