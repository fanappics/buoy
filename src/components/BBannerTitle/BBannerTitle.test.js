import { mount } from 'vue-test-utils'
import BBannerTitle from './BBannerTitle'

describe('BBannerTitle', () => {
  it('should render banner title', () => {
    const wrapper = mount(BBannerTitle)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
