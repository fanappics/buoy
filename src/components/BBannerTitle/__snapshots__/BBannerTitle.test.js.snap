// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BBannerTitle should render banner title 1`] = `<span class="b-banner-title"></span>`;
